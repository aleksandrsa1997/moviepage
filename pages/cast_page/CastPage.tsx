import React from 'react';
import STFlex  from 'src/reactlibs/STFlex/STFlex';
import CustomSpinner from 'src/reactlibs/CustomSpinner';

const CastPage = () => (
    <STFlex data-at='wrapper_cast-page'>
        <CustomSpinner
            size={150}
            color='primary'
            variant='indeterminate'
            thickness={5}
            disableShrink
        />    
    </STFlex>
);

export default CastPage;
