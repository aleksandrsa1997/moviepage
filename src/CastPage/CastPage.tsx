import STFlex from 'src/reactlibs/STFlex';
import CustomSpinner from 'src/reactlibs/CustomSpinner';

const CastPage = () => (
    <STFlex data-at='wrapper_cast-page'>
        <CustomSpinner
            size={150}
            color='primary'
            variant='indeterminate'
            thickness={5}
            disableShrink
        />
    </STFlex>
);

export default CastPage;