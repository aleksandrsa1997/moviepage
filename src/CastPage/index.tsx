// @ts-ignore
import CastPage from './CastPage';

const IndexPage = () => (
    <CastPage/>
)

export default IndexPage;