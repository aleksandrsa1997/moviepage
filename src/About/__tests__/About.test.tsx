import React from 'react';
import CastPage from 'src/CastPage';

describe('About snapshot', () => {
    it('should correct renderer', () => {
        const component = renderer(<CastPage />);

        expect(component).toMatchSnapshot();
    });
});