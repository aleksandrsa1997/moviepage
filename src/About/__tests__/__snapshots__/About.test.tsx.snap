// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About snapshot should correct renderer 1`] = `
<div
  className="STFlex-m28u7z-0 qXibz"
  data-at="wrapper_cast-page"
>
  <div
    className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
    role="progressbar"
    style={
      Object {
        "height": 150,
        "width": 150,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
        cx={44}
        cy={44}
        fill="none"
        r={19.5}
        strokeWidth={5}
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
